.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,12 @@ More in-depth discussion of the change or implementation.
 
 Trello, screenshots, external resources?
 
+# Acceptance criteria ✅
```

Comment:
Can we move those emojis to the beginning of a line (instead of the end)? The alignment will look much nicer!